docs/10.0/guides/getting-started/introduction.md

```diff
@@ -14,7 +14,7 @@ Welcome 👋&nbsp; to Handsontable's Guides. Here you will learn how Handsontabl
 ## Get started with sample apps
 
 <div class="row-items-container">
-    <a href="/docs/10.0/hello-world" class="row-item">
+    <a href="https://handsontable.com/demo" class="row-item">
```

Comment:
I think that this should not take the reader out of the docs. Rather, similarly to the framework logos, this can go to the simplest example in plain JavaScript. How about `/docs/10/binding-to-data`

Reply:
@warpech - ok, done: /docs/binding-to-data